infrastructure_overview/service_types.md

```diff
@@ -250,17 +250,17 @@ This method will be called to invoke your service.
 
 | Parameter name | Optional | Description |
 --- | --- | :--- |
-| `sessionToken` | yes | Session token to be used throughout the workflow |
-| `serviceID` | yes | The ID assigned to this service by the execution engine, provided automatically |
-| `extraParameters` | yes | Needs to be connected to the "extraParameters" workflow input |
+| `sessionToken` | no | Session token to be used throughout the workflow |
+| `serviceID` | no | The ID assigned to this service by the execution engine, provided automatically |
+| `extraParameters` | no | Needs to be connected to the "extraParameters" workflow input |
 | ... |  | |
 | `your parameters` | no | |
 
 ##### Outputs
 
 | Parameter name  | Optional | Description |
 --- | --- | :--- |
-| `status_base64` |  no | The status of your workflow (`COMPLETED`, `UNCHANGED`, or a base64-encoded string) |
+| `status_base64` |  no | The status of your workflow (a base64-encoded string) |
```

Comment:
Why are "COMPLETED" and "UNCHANGED" being removed? Is this not correct anymore?

Reply:
As this is the very first status of the component it may not yet be COMPLETED or UNCHANGED. I will add this to the asynchronous "startService"-method as well, and adapt the descriptions of the "getServiceStatus" and "serviceExecutionFinished" to include those values again.